packages/e2e-test-utils/src/switch-block-inspector-tab.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Clicks on the block inspector tab button with the supplied label and waits
+ * for the tab switch.
+ *
+ * @param { string } label Aria label to find tab button by.
+ */
+export async function switchBlockInspectorTab( label ) {
+	const tabButton = await page.$(
+		`.block-editor-block-inspector__tabs button[aria-label="${ label }"]`
+	);
+
+	if ( tabButton ) {
```

Comment:
This should probably throw an error rather than skipping the steps silently if the tab button is not there for debugging?

Reply:
✅ Done in https://github.com/WordPress/gutenberg/pull/47097